README.md

```diff
@@ -1,10 +1,11 @@
-# DAOStack Alchemy - Creating DAOs for an Emergent Future
+# DAOstack Alchemy - Creating DAOs for a Collective Intelligence
 
 ## What is Alchemy?
 
-DAOStack Alchemy is a browser application that enables anyone with an account on the Ethereum blockchain, and STK tokens to spend, to easily create and work with DAOs on Ethereum that run on predefined, configurable smart contracts in a network of organizations, collaborators and investors that collaborate on the blockchain.
+DAOstack Alchemy is a browser application that enables anyone with an account on the Ethereum blockchain, and STK tokens to spend, to easily create and work with DAOs on Ethereum that run on predefined, configurable smart contracts in a network of organizations, collaborators and investors that collaborate on the blockchain.
```

Comment:
> DAOstack Alchemy is a browser application that enables anyone with an account on the Ethereum blockchain, and STK tokens to spend, to easily create and work with DAOs on Ethereum that run on predefined, configurable smart contracts in a network of organizations, collaborators and investors that collaborate on the blockchain.

I will revised this long sentence. It is long and not so accurate.
Is that a Vanilla or Alchemy repo ?
STK - > GEN

Reply:
Good catch on GEN and Vanille.  As for the run-on sentence, I'm having a hard time making it shorter.  But I edited it a little to make it flow a little better.  It works for me, but maybe I'm more accustomed to reading long sentences...